### ISSUE TITLE
Prevent submission of drives with duplicate measurements

### ISSUE THREAD

**Issue by auspicacious** [Safecast/safecastapi#595]
Right now, if a drive has already been uploaded to API, and the exact same drive file is uploaded a second time, it will be rejected with an appropriate error message.

If some measurements are removed from the file, and the file is resubmitted, it will not be rejected immediately, but instead the submission will enter a limbo state because there are no new measurements to process; this is confusing and adds noise to the database. Such files should be rejected.

Another question would be what to do if there is a mix of old and new data in the file. We should at least note this fact explicitly in the UI.

See also #467

**Comment by seanbonner** [comment]
Correct. The most frequent cause of the last point is that bGeigies make one log file per day, so sometimes people will collect data, upload it, then begin measuring again on the same day and the bGeigie will just amend the existing log file with the new data rather than making a new log. So then they upload the file again, perhaps several days later, the file will show something like:

Number Of Lines
10652
Number Of Measurements
52

Which results in confusion for the uploader as well as the moderators as they have to see if the other measurements are preexisting or invalid or something else. In these cases it would be ideal to have some notification like: "10600 duplicate measurements ignored"
